src/Halogen/Svg/Indexed.purs

```diff
@@ -78,6 +78,7 @@ type SVGg = GlobalAttributes
 type SVGpath = MarkerAttributes (GlobalAttributes
   ( d :: String
   , transform :: String
+  , strokeWidth :: Number
```

Comment:
Could you remove this one line? Once #12 gets merged, it'll be included here anyway and that means there won't be a merge conflict if I merge this one first.

Reply:
done!